docs/source/install/min-prod-hw.rst

```diff
@@ -30,3 +30,40 @@ Prometheus uses more memory when querying over a longer duration (e.g. looking a
 
 For Prometheus alone, you should have 60MB of memory per core in the cluster and it would use about 600MB of virtual memory per core.
 Because Prometheus is so memory demanding, it is a good idea to add swap, so queries with a longer duration would not crash the server.
+
+.. raw:: html
```

Comment:
@dgarcia360 is this a good way to embed script in a page, or is there a more idiomatic way?